src/platforms/dart/configuration/integrations/index.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: Integrations
+sidebar_order: 30
+description: "Learn more about how integrations extend the functionality of our SDK to cover common libraries and environments automatically."
+---
+
+[HTTP Integration]()
+
+    Learn more about the Sentry HTTP integration for the Dart SDK.
```

Comment:
This should be a `PageGrid` I believe, looking at the Android docs https://github.com/getsentry/sentry-docs/edit/master/src/platforms/android/configuration/integrations/index.mdx

Reply:
I was trying to avoid having a page with one bullet point, but other formatting options weren't working so I've used the PageGrid component.